profile/README.md

```diff
@@ -56,6 +57,20 @@ alt="Service Mesh Landscape Ecosystem" align="left"/></a>
 The <a href="https://layer5.io/landscape">Service Mesh Landscape</a> is a comprehensive collection of prominent service mesh projects and offerings laid out in contrast to one another.
 <br />
 <br />
+</p>
+<br />
+<br />
+<h2><a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a></h2>
+<a href="https://meshery.io"><img src="https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/.github/readme/images/service-mesh-pattern.png"
+style="margin:10px;" width="125px" 
+alt="Meshery - Service Mesh Patterns" align="left" /></a>
+
+<a href="https://github.com/service-mesh-patterns">Service Mesh Patterns</a> help you get the most out of Linkerd, Consul, App Mesh, Istio, Kuma, Open Service Mesh or <a href="https://layer5.io/service-mesh-landscape">any service mesh</a>, including those using Envoy and use of WebAssembly. Each pattern can be used as a template and is customizable.
+<br /><br /><br />
```

Comment:
Were you able to get the "help wanted" shield in here, too?

Reply:
No, I did not add help wanted shield wlll add asap. Can you please provide the hyperlink to be associated with it

Reply:
@Shivanshu97i https://img.shields.io/github/issues-search?label=help%20wanted&query=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+repo%3Aservice-mesh-patterns%2Fservice-mesh-patterns this should work.

Reply:
Thanks, Will add this immediately

Reply:
@Shivanshu97i do attach a screenshot of the final preview.

Reply:
![Issue 5](https://user-images.githubusercontent.com/62894515/134773621-25478328-39e8-4b4f-bff1-3a0e20726a72.jpg)
Sorry, The images are not loading don't know why but the final preview looks something like this